docs/config.rst

```diff
@@ -359,6 +359,10 @@ A stack has the following keys:
 **region**:
   (optional): If provided, specifies the name of the region that the
   CloudFormation stack should reside in.
+**profile**:
```

Comment:
What happens if this is provided, and region is provided?  Pretty sure you can provide a region as part of a profile, so curious which wins out (and it should probably be documented)

Reply:
I'll add this to the docs. If both `profile` and `region` are provided, the region that's used in the config overrides the region in the `profile`.